src/sqltest/parts/notebook/model/notebookModel.test.ts

```diff
@@ -141,21 +141,21 @@ suite('notebook model', function (): void {
 	});
 
 	// test('Should throw if model load fails', async function(): Promise<void> {
-	//     // Given a call to get Contents fails
-	//     let error = new Error('File not found');
-	//     let mockContentManager = TypeMoq.Mock.ofType(LocalContentManager);
-	//     mockContentManager.setup(c => c.getNotebookContents(TypeMoq.It.isAny())).throws(error);
-	//     notebookManagers[0].contentManager = mockContentManager.object;
-
-	//     // When I initalize the model
-	//     // Then it should throw
-	//     let model = new NotebookModel(defaultModelOptions);
-	//     should(model.inErrorState).be.false();
-	//     await testUtils.assertThrowsAsync(() => model.requestModelLoad(), error.message);
-	//     should(model.inErrorState).be.true();
+	// 	// Given a call to get Contents fails
```

Comment:
Let's plan & prioritize reenabling tests once we have stability resolved.

Reply:
Will do.